### PR TITLE
Add: admin's Name

### PR DESCRIPTION
**Pull request by stephenbech**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ function Header() {
 
      
       <Isme>
+        <h4 className='Adminsname'>Stephen Nzubechukwu</h4>
+        {/* <h5>Church Admin</h5> */}
         <UserImg src="./images/admin.jpg" />
```
Comment (marveldc08) [inline src/components/Header.js:28]: Avoid static URLs

**Review by marveldc08 — CHANGES_REQUESTED** [review]

File: src/components/Header.js (hunk 1 above)
Comment (stephenbech) [reply src/components/Header.js:28]: moved to images configuration file

**Comment by stephenbech** [comment]
Done

**Comment by stephenbech** [comment]
Done

**Pull request merged**